docs/chapter_heap/heap.md

```diff
@@ -518,34 +550,34 @@ comments: true
 顾名思义，**从顶至底堆化的操作方向与从底至顶堆化相反**，我们比较根结点的值与其两个子结点的值，将最大的子结点与根结点执行交换，并循环以上操作，直到越过叶结点时结束，或当遇到无需交换的结点时提前结束。
 
 === "Step 1"
-    ![heap_poll_step1](heap.assets/heap_poll_step1.png)
```

Comment:
Same as above

Reply:
I'm very sorry about this, here it's the vscode plugin that automatically typesets the code and I didn't notice the error

Reply:
No problem. Let's revert it back